repo/blob/storage.go

```diff
@@ -62,6 +65,9 @@ type PutOptions struct {
 	RetentionMode   string
 	RetentionPeriod time.Duration
 
+	// if true, PutBlob will fail with ErrBlobAlready exists if a blob with the same ID exists.
```

Comment:
nit; `ErrBlobAlready exists` -> `ErrBlobAlreadyExists`

Reply:
Done, thanks for catching that.